CIP-????/README.md

```diff
@@ -0,0 +1,294 @@
+Category: Plutus
+author: Michał J. Gajda (mjgajda@migamake.com)
+Implementors: []
+Discussions-To: https://github.com/cardano-foundation/CIPs/pull/441
+Comments-Summary: 
+Comments-URI:  
+Status: Proposed
+Type: Standards Track
+Created: 2023-01-19  
+License: Apache-2.0
+---
+## Simple Summary
+
+Support post-quantum signatures (Dilithium+CRYSTALS, FALCON) as recommended by NIST standardization effort in Plutus Core;
+allow validation of such signatures as builtins.
+NIST plans to finalize standardization in 2023/2024 time frame, with US executive branch recommending
+all projects to plan for post-quantum resistance by mid-2023.
+We should prepare Cardano Blockchain for migration.
+
+## Abstract
+
+Provides a way of verifying post-quantum signatures as recommended by NIST in Plutus Core.
+New builtins will allow use of these signatures within smart contracts. These builtins work over
+`BuiltinByteString`s.
+
+## Motivation
+
+NIST estimates that pre-quantum cryptographic algorithms may be no longer safe
+by 2035. However, the smart contracts have long development and lifetime,
+so migration to post-quantum cryptography algorithms should start several
+years before they would become insecure.
+This summer NIST has recommended three PQC signature algorithms
+for standardization, publishing their reference implementations,
+and finishing initial comment period.
+
+Supporting PQC signatures will bring the following benefits:
+
+* smart contracts may use future standard algorithms today;
+* early support for PQC will simplify future PQC migration, if needed;
+* future PQC migration would be smooth and require less changes;
+* support for PQC may increase industry acceptance for long running projects;
+* users that want their wallets to survive over ten years may
+  use smart contracts verifying multiple algorithms today;
+* Cardano will become first blockchain to verify PQC standard signatures,
+  thus getting first-mover advantage.
+
+## Use cases
+
+Adding multiple PQC signature algorithms will allow the following:
+
+* opt-in migration of wallets and smart contracts to safer cryptographic protocol
+* pay-as-you-go for longer signatures required for post-quantum resistance
+* opt-in resistance to _cryptocalypse_ - surprise breakage of a single major encryption algorithm -
+  by using multiple signature algorithms for redundancy.
+* compliance with US government regulations past July 2023, when all agencies
+  are expected to plan post-quantum migration.
+
+## Specification
+
+[`DSIGN`]() class instance should be provided for the post-quantum signature algorithms.
+
+### Plutus API
+
+Two new builtin functions would be provided:
+
+* A verification function for both pre-quantum and post-quantum resistant
+  signature schemes:
+     - CRYSTALS(DILITHIUM) signatures;
+     - FALCON signatures;
+     - SPHINCS signatures; and
+     - SEQUOA
+* Extensible interface for signature verification functions.
```

Comment:
Same comment as in the CPS: it's not clear why this is entangled with the PQC stuff.